doc/flame/effects.md

```diff
@@ -111,6 +111,7 @@ offset is relative to the current position of the target:
 ```
 
 ```dart
+
```

Comment:
Why are you adding these lines in this PR?

Reply:
i think its added by ide format.

Reply:
Your IDE seems to have some trouble with formatting, these lines should not be added. The other formatting fixes look fine though.

Reply:
Ah you fixed it, good!